### ISSUE TITLE
Suggestion: Enable dark theme manually 

### ISSUE THREAD

**Issue by cpasoft** [BiGilSoft/WindowTop#185]
Allow to set the black theme manually.

**Comment by gileli121** [comment]
@cpasoft 
I understand your request.
But do you have a more valid reason why this is important?
Most of the time you don't use the settings UI.
You use the toolbar or the tray menu.

I think that your main reason for asking for this feature is because you use the tray menu a lot?

**Comment by cpasoft** [comment]
Yes, it is.. the tray menu on gray is not very aesthetic. The main program it's good enough on day mode...

**Comment by cpasoft** [comment]
Here you can see the "day mode" menu vs "night mode" menu... A lot of better the second one...

Have a look:
![imagen](https://user-images.githubusercontent.com/29692062/148644718-cc06b6d3-300f-47b3-97eb-f097a128f1f1.png)
![imagen](https://user-images.githubusercontent.com/29692062/148644723-09107810-8f81-442b-af6f-acb33fe7cdf2.png)

**Comment by cpasoft** [comment]
Of course, is a personal preference... :-)

**Comment by gileli121** [comment]
OK, I did not thought like that.
For me both are fine. 
But you are right, it is personal preference.

I will add this option in the next releases..
For now I am going to release what we got so far as stable release

**Comment by cpasoft** [comment]
Thank you Gileli... It is not an urgent need. :-)

**Comment by gileli121** [comment]
@cpasoft 
Do you think that the new default settings are fine?
I think that maybe this option
![image](https://user-images.githubusercontent.com/17680514/148678120-e09a49ac-3507-4d96-87b3-9f10c825cb87.png)

Should be disabled by default and not enabled

**Comment by cpasoft** [comment]
But this is not related with this topic, isn't it??...

Anyway, answering your question, yes, I guess this option should be disabled by default, but is not important.
